pex/resolve/target_options.py

```diff
@@ -61,16 +61,12 @@ def register(
         ),
     )
 
-    current_interpreter = PythonInterpreter.get()
-    program = sys.argv[0]
     singe_interpreter_info_cmd = (
```

Comment:
Fix the typo to `single_` while you're here?

Reply:
Fixed.